advanced/events.md

```diff
@@ -157,6 +157,7 @@ The following events are provided by the {{ site.productname }} editor.
 
 The following plugins provide events.
 
+  - [Advanced Tables events](#advancedtablesevents)
```

Comment:
This doesn't appear to match up with the link above

Reply:
it should match the heading on this page, which it does.

To clarify, on a plugin page, we would/do use the heading "Events", on this page, we use "<plugin name> events"